Resources/Prototypes/Entities/Items.yml

```diff
@@ -76,8 +76,15 @@
   id: FlashlightLantern
   description: They light the way to freedom
   components:
+  - type: HandheldLight
   - type: Sprite
-    texture: Objects/Flashlight.png
+    sprite: Objects/lantern.rsi
+    layers:
+    - state: lantern_off
```

Comment:
The reason the outlines don't work is because their implementation is pretty terrible and they modify the shader of the first layer.

If you wanna fix it you can make this use one layer and just change the state instead of toggling visibility.